swap/swap.go

```diff
@@ -270,35 +305,30 @@ func (s *Swap) createCheque(peer enode.ID) (*Cheque, error) {
 	return cheque, err
 }
 
-// PeerBalance returns the balance for a given peer
-func (s *Swap) PeerBalance(peer enode.ID) (int64, error) {
-	s.lock.RLock()
-	defer s.lock.RUnlock()
-	var peerBalance int64
-	var err error
-
-	// load balance from memory
-	peerBalance, keyExists := s.balances[peer]
-	if !keyExists {
-		// if not in memory, try to load balance from disk
-		err = s.stateStore.Get(peer.String(), &peerBalance)
+// Balance returns the balance for a given peer
+func (s *Swap) Balance(peer enode.ID) (peerBalance int64, err error) {
```

Comment:
the function signatures are not consistent - in some cases you have named return values, you set them in the code (not a very good practice IMO), then `return` when the function runs to the end. in others, you return within the function (e.g. `Balances()` below)

Reply:
Good point, I've removed the named parameters.

Reply:
Most of these issues were probably introduced by me in the past.